packages/sanity/src/core/store/_legacy/document/document-pair/documentEvents.ts

```diff
@@ -11,8 +11,13 @@ import {memoizeKeyGen} from './memoizeKeyGen'
 // A stream of all events related to either published or draft, each event comes with a 'target'
 // that specifies which version (draft|published) the event is about
 export const documentEvents = memoize(
-  (client: SanityClient, idPair: IdPair, typeName: string): Observable<DocumentVersionEvent> => {
-    return memoizedPair(client, idPair, typeName).pipe(
+  (
+    client: SanityClient,
+    idPair: IdPair,
+    typeName: string,
+    serverActionsEnabled?: boolean,
```

Comment:
this calls for a tiny bit of caution I guess, since the events stream is memoized on a key based on all the other arguments, a change in the enabled status will not give a cache miss, but rather keep returning the one with the old flag

Reply:
a workable fix for now is probably to include it with memoizeKeygen

Reply:
ah i was wondering if I should have done that. Added now!